geode-core/src/main/java/org/apache/geode/distributed/internal/LocatorLoadSnapshot.java

```diff
@@ -50,7 +50,8 @@
 
   private final Map<ServerLocation, String[]> serverGroupMap = new HashMap<>();
 
-  private final Map<String, Map<ServerLocation, LoadHolder>> connectionLoadMap = new HashMap<>();
+  private final Map<String, Map<ServerLocationAndMemberId, LoadHolder>> connectionLoadMap =
+      new HashMap<>();
 
   private final Map<String, Map<ServerLocation, LoadHolder>> queueLoadMap = new HashMap<>();
```

Comment:
This map needs to change for the same reasons the `connectionLoadMap` needs changing.

Reply:
done